internal/app/app.go

```diff
@@ -140,6 +141,25 @@ func apiPublished(cfg *ServerConfig) echo.HandlerFunc {
 	}
 }
 
+func apiBasicAuth(cfg *ServerConfig) echo.HandlerFunc {
```

Comment:
I created this PR's branch from another branch which approved by @rot1024  but not by @KaWaite . I expect the branch will be merged soon. The difference is only this file.